### DISCUSSION TITLE
[Android] Renaming Mapbox to MapLibre across the codebase

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/maplibre-native#688] [General]
The resource identifiers prefix was changed from `mapbox_` to `maplibre` in #647.

Since some resources are public this is a breaking change (albeit one that is relatively easy to fix for library users).

Do we want to add other Mapbox -> MapLibre name changes in the next release, so users only have to deal with this once?

Related: #318

**Comment by ntadej** [comment]
We could but then we have to bump the major version.

**Reply by louwers** [reply]
We need to bump it anyway because of #647.

**Reply by ntadej** [reply]
Then I vote in favour of the rename.

The question is also when do we need a new release. We could wait a bit to accumulate more breaking changes.

**Reply by boldtrn** [reply]
How about pre releases until we are ready for the next major?

**Reply by birkskyum** [reply]
@ntadej , do you have anything particular in mind you'd like to go into the next major? I'm personally not a fan of all this "accumulating many breaking change in one release", as it makes it much harder for users to upgrade and seems more like a marketing concern. In my experience, taking small steps between versions make the upgrade easier.

**Reply by lseelenbinder** [reply]
I generally agree with @birkskyum, unless we find ourselves bumping major versions every other week. :smile: I'd rather keep a steady stream of releases as we get fixes in.

**Reply by ntadej** [reply]
Then I propose we do a big rename and bump the release and then try to keep compatibility for a while.
